modules/dnn/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ ocv_add_dispatched_file_force_all("layers/layers_common" AVX AVX2 AVX512_SKX)
 ocv_add_module(dnn opencv_core opencv_imgproc WRAP python java js)
 
 ocv_option(OPENCV_DNN_OPENCL "Build with OpenCL support" HAVE_OPENCL AND NOT APPLE)
+ocv_option(OPENCV_DNN_TENGINE "Build with Tengine support" HAVE_TENGINE)
```

Comment:
> OPENCV_DNN_TENGINE

Probably not needed at all.
OpenCL is OpenCV-wide stuff (so we want to control this through `OPENCV_DNN_OPENCL`), but tengine is not.

Reply:
done